packages/cactus-plugin-satp-hermes/package.json

```diff
@@ -134,6 +134,7 @@
     "kubo-rpc-client": "3.0.1",
     "npm-run-all": "4.1.5",
     "openzeppelin-solidity": "3.4.2",
+    "pg": "^8.13.1",
```

Comment:
please fix the pg version

Reply:
for types too

Reply:
I didn't understand, I mean adding a stable version of it?

Reply:
removing the caret